django-backend/fecfiler/f3x_summaries/migrations/0001_initial.py

```diff
@@ -0,0 +1,148 @@
+# Generated by Django 3.2.11 on 2022-02-10 22:51
+
+from django.db import migrations, models
+
+
```

Comment:
I was thinking that the migrations files were like the files in node_modules in that they are not actually saved to the repo but build by a "make migrations" call during the build/deploy process. Let's discuss.